install_config/aggregate_logging.adoc

```diff
@@ -102,309 +88,406 @@ deployed. To control component placement, specify node selectors per component t
+when `openshift_logging_use_ops` is set to `true`.
+
+|`openshift_logging_es_ops_client_key`
+|Equivalent to `openshift_logging_es_client_key` for Ops cluster
+when `openshift_logging_use_ops` is set to `true`.
+
+|`openshift_logging_es_ops_cluster_size`
+|Equivalent to `openshift_logging_es_cluster_size` for Ops cluster
+when `openshift_logging_use_ops` is set to `true`.
+
+|`openshift_logging_es_ops_cpu_limit`
+|Equivalent to `openshift_logging_es_cpu_limit` for Ops cluster
+when `openshift_logging_use_ops` is set to `true`.
+
+|`openshift_logging_es_ops_memory_limit`
+|Equivalent to `openshift_logging_es_memory_limit` for Ops cluster
+when `openshift_logging_use_ops` is set to `true`.
+
+|`openshift_logging_es_ops_pv_selector`
+|Equivalent to `openshift_logging_es_pv_selector` for Ops cluster
+when `openshift_logging_use_ops` is set to `true`.
 
-|*_MODE_* (default: *install*)
-| Mode to run the deployer in; one of `install`, `uninstall`, `reinstall`, `upgrade`, `migrate`, `start`, `stop`.
+|`openshift_logging_es_ops_pvc_dynamic`
+|Equivalent to `openshift_logging_es_pvc_dynamic` for Ops cluster
+when `openshift_logging_use_ops` is set to `true`.
+
+|`openshift_logging_es_ops_pvc_size`
+|Equivalent to `openshift_logging_es_pvc_size` for Ops cluster
+when `openshift_logging_use_ops` is set to `true`.
+
+|`openshift_logging_es_ops_pvc_prefix`
+|Equivalent to `openshift_logging_es_pvc_prefix` for Ops cluster
+when `openshift_logging_use_ops` is set to `true`.
+
+|`openshift_logging_es_ops_recover_after_time`
+|Equivalent to `openshift_logging_es_recovery_after_time` for Ops cluster
+when `openshift_logging_use_ops` is set to `true`.
+
+|`openshift_logging_es_ops_storage_group`
+| Equivalent to `openshift_logging_es_storage_group` for Ops cluster
+when `openshift_logging_use_ops` is set to `true`.
+
+|`openshift_logging_es_ops_nodeselector`
+|A node selector that specifies which nodes are eligible targets
+for deploying Elasticsearch instances. This can be used to place
+these instances on nodes reserved or optimized for running them.
+For example, the selector could be `node-type=infrastructure`. At least
+one active node must have this label before Elasticsearch will deploy.
+
+|`openshift_logging_kibana_ops_nodeselector`
+|A node selector that specifies which nodes are eligible targets
+for deploying Kibana instances.
+
+|`openshift_logging_curator_ops_nodeselector`
+|A node selector that specifies which nodes are eligible targets
+for deploying Curator instances.
 |===
 
-Running the deployer creates a deployer pod and prints its name. Wait until the
-pod is running. This can take up to a few minutes for {product-title} to
-retrieve the deployer image from the registry. Watch its process with:
+//. Create a xref:../dev_guide/secrets.adoc#dev-guide-secrets[secret] to provide security-related files to the deployer. Providing the secret is optional, and the objects will be randomly generated if not supplied.
```

Comment:
@bmcelvee Added you to https://trello.com/c/icrVWXjo/ which addresses this section.  Can you make sure to follow up with me to ensure we get it addressed